src/near/types/actions.rs

```diff
@@ -372,53 +372,141 @@ mod tests {
     use crate::near::types::public_key::ED25519PublicKey;
     use near_sdk::serde_json;
 
-    fn get_actions() -> Vec<Action> {
+    use near_primitives::action::{
+        Action as NearPrimitiveAction, AddKeyAction as NearPrimitiveAddKeyAction,
```

Comment:
I guess it's ok. But also consider something like:
```
use near_primitives::action as primitives;
```

and use like:
```
primitives::DeleteAccountAction
```

Reply:
It is done similarly in others tests, so I decided to stick to the style